lib/MusicBrainz/Server/Data/WikiDoc.pm

```diff
@@ -50,10 +50,11 @@ sub _fix_html_links
             $node->replace_with($node->content_list);
         }
     }
-    # if this is not a link to the wikidocs server, don't mess with it.
-    elsif ($href =~ m,^(?:https?:)?//$wiki_server,)
+    # If this is not a link to the wikidocs server, don't mess with it.
+    # Relative links inside the wiki should be wiki links, so convert them too.
+    elsif ($href =~ m,^(?:\/|(?:https?:)?//$wiki_server),)
     {
-        $href =~ s,^(?:https?:)?//$wiki_server/?,/doc/,;
+        $href =~ s,^(?:\/|(?:https?:)?//$wiki_server/?),/doc/,;
```

Comment:
(No need to escape `/` because `,` is being used as the delimiter)
```suggestion
        $href =~ s,^(?:/|(?:https?:)?//$wiki_server/?),/doc/,;
```